pkgs/by-name/nb/nbfc-linux/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  callPackage,
+  lib,
+  stdenv,
+  fetchurl,
+  nixos,
+  testers,
+  hello,
+  fetchgit,
+}:
+stdenv.mkDerivation (finalAttrs: let
+  buildFlags = ["PREFIX=$(out)" "confdir=/etc"];
+  installFlags = ["PREFIX=$out" "confdir=$out/etc"];
+in rec {
+  inherit buildFlags;
```

Comment:
Please move this attributes (see next comment). `rec` and `buildFlags` are not needed.

Reply:
the buildFlags are needed, without them nbfc looks into /usr/local/share/nbfc/configs/  instead of $out/etc/nbfc/configs